raspberry/DiscordBot/README.md

```diff
@@ -57,10 +63,27 @@ Then, just run the script
 python main.py
 ```
 
+
+### How to run the client on Start_up
+
+After the virtual environment and dependencies have been installed as described above, a script can be used to atuomatically run the client on start_up.
+
+To do this, the `run_DiscordBot_on_start_up.sh` can be added to the users crontab by:
+
+```
+crontab -e
+```
+Then add the following:
+
+```
+@reboot sleep 30 && ~/bzzz/raspberry/DiscordBot/run_DiscordBot_on_start_up.sh
+```
+
+
 ### Chatting with the bot
 
 Firstly, you need to join our private Discord server. Ask one of the team members for an invitation.
 
 In the list of participants you can see which agents (bots) are live. Type `.ip`* to get their IPs. To format the output using markdown, use `.ip p` (here 'p' stands for 'pretty')
 
-> * It is not case-sensitive, so `.IP` works as well
\ No newline at end of file
+> * It is not case-sensitive, so `.IP` works as well
```

Comment:
Have a look [here](https://github.com/QUB-ASL/bzzz/tree/feature/148-run_on_start_up/raspberry/DiscordBot#chatting-with-the-bot). The asterisk shows as a bullet.

Reply:
is this meant to be a footnote? if so we can do [this](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#footnotes)